index.html

```diff
@@ -78,6 +78,15 @@ <h4 class="subheading">Activities: </h4>
           </div>
         </div>
       </section>
+      <section class="" id="Favorites">
```

Comment:
If you don't plan to give this a `class`, remove the `class` attribute

Reply:
Class will toggle on and off between views.